src/Pages/ProductDetail/ProductDetail.scss

```
@@ -0,0 +1,369 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&display=swap');
+
+.contents {
+  height: 100%;
+  width: 1450px;
+  padding: 20px 0px 0px;
+  align-items: center;
+  margin: 0 auto;
+
+  .sub_content {
+    margin: 0px auto;
+    padding: 40px;
+
+    .item_photo_info_sec {
+      display: flex;
+      width: 1200px;
+      height: 586px;
+      padding: 0px 40px;
+      margin: 0 auto;
+
+      .item_photo_big {
+        width: 500px;
+      }
+    }
+    .item_detail_cont {
+      margin-left: 70px;
+      width: 556px;
+      height: 585px;
+      font-weight: bold;
+
+      dt {
+        width: 120px;
+        height: 27px;
+        color: #717171;
+        margin: 0 10px 0 5px;
+        font-size: 16px;
+      }
+
+      .item_delivery {
+        margin-top: 8px;
+      }
+    }
+
+    .item_detail_list {
+      margin: 10px 0px 20px;
+      padding: 0px 0px 10px;
+    }
+    h3 {
+      font-size: 35px;
+      font-weight: bold;
+      display: block;
+      font-family: ‘Noto Sans KR’, sans-serif;
+      margin-top: 10px;
+      padding: 0px 0px 30px;
+    }
+
+    dl {
+      display: flex;
+    }
+
+    .item_price {
+      font-size: 21px;
+    }
+
+    .item_detail_cont {
+      height: 585px;
+
+      .original_price,
+      .delivery_fee_free,
+      .limit_time {
+        font-weight: normal;
+      }
+
+      .delivery_fee_free {
+        margin-bottom: 10px;
+      }
+
+      table {
+        background-color: #f5f5f5;
+        height: 59px;
+
+        .cart_prdt_name {
+          width: 330px;
+          vertical-align: middle;
+
+          .product_name {
+            font-size: 13px;
+            margin-left: 20px;
+          }
+        }
+
+        td {
+          vertical-align: middle;
+        }
+
+        .prdt_count {
+          width: 105px;
+          display: flex;
+
+          .count {
+            display: flex;
+            justify-content: center;
+          }
+          .count_box {
+            width: 35px;
+            height: 32-33px;
+            border: 1px solid lightgrey;
+            text-align: center;
+            margin-top: 8px;
+          }
+        }
+
+        .up_goods_cnt {
+          margin-top: 8px;
+          height: 20px;
+        }
+
+        .down_goods_cnt {
+          height: 20px;
+        }
+        .up_goods_cnt,
+        .down_goods_cnt,
+        .del_goods_cnt {
```

해당하는 요소에 동일한 class명을 한 개 더 부여해주면 선택자를 세 개 적을 필요없이 세 요소에 모두 동일하게 부여된 class명을 작성해주시면 됩니다.

```suggestion
        .grayColor {
          color: gray
        }
```
```jsx
  <div class="up_goods_cnt grayColor" />
```